protocol/v2/qbft/instance/instance.go

```diff
@@ -22,7 +22,9 @@ type Instance struct {
 
 	processMsgF *spectypes.ThreadSafeF
 	startOnce   sync.Once
-	StartValue  []byte
+
+	forceStop  bool
```

Comment:
Should we make it atomic?

Reply:
currently instance isn't thread safe. so there's no reason to protect `forceStop` only.